ldms/src/ldmsd/ldmsd_request.c

```diff
@@ -6226,6 +6228,7 @@ void stream_xprt_term(ldms_t x)
 	struct __RSE_key_s *key = (void*)_buff;
 	int rc;
 
+	key->addr_len = sizeof(key->addr);
```

Comment:
@tom95858 Without this line, `key->addr_len` is 0 and, thus, `ldms_xprt_sockaddr` would return an error.  The function `stream_xprt_term` removes the client from the client list when the connection to the client is disconnected. Since it cannot get the address, it won't remove the client from the list upon disconnection.